### ISSUE TITLE
way to use dvd repo then update it using aptly from mirror

### ISSUE THREAD

**Issue by ZamanOof** [aptly-dev/aptly#982]
Hello,
are there way as mentioned to use local repo then updated it from mirror?

**Comment by r4co0n** [comment]
`aptly` manages a single pool of packages, so if you import the packages on your optical disc, they will be recognized as already fetched on subsequent remote mirror synchronizations.

**Comment by ZamanOof** [comment]
yes sure, I was thinking put online locally then change to online mirror(web mirror) and it would be nice to document it
